.github/workflows/ecr-publisher-release.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       - id: version
         name: Determine Container Tag
         run: |
-          IMAGE_TAG="${$(jq -r .version version.json)#v}"
```

Comment:
This line strips `v` from the beginning of the value associated to `version` key in JSON object contained by `version.json` file.

The cause of build failure must be something else, as testing this locally on my bash session it works; Try running: ` IMAGE_TAG="${$(jq -r .version version.json)#v}" && echo $IMAGE_TAG` in root of `storetheindex`

Reply:
Try creating a file with the following contents:
```
#!/bin/bash

IMAGE_TAG="${$(jq -r .version version.json)#v}"
echo "Using image tag: ${IMAGE_TAG}"
```

If you execute it you'd get the bad substitution error. Removing surrounding`${` resolves it. There must be some other way to remove `v` then

Reply:
Ok changed it to strip out `v` in two steps seems to be working now